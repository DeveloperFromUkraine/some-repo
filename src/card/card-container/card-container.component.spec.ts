import { ComponentFixture, TestBed } from "@angular/core/testing";
import { CardContainerComponent } from "./index";
import { ComponentTest } from '../../../test/test-bed/component';
import { MatCardModule } from "@angular/material";

describe('CardContainerComponent', () => {
    let fixture: ComponentFixture<CardContainerComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([MatCardModule], [CardContainerComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(CardContainerComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});