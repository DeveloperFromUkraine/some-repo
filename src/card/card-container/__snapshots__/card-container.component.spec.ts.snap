// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainerComponent should match snapshot 1`] = `
<ign-card-container>
  <mat-card
    class="container mat-card"
  >
    
    
    
  
  </mat-card>
  
</ign-card-container>
`;
